frontend/src/views/Game.tsx

```diff
@@ -10,20 +16,45 @@ type LocationState = {
 function GamePage() {
   const location = useLocation<LocationState>();
   const [room, setRoom] = useState<Room | null>(null);
+  const [problems, setProblems] = useState<Problem[] | null>(null);
+  const [error, setError] = useState<string | null>(null);
 
   // Called every time location changes
   useEffect(() => {
     if (location && location.state && location.state.room) {
       setRoom(location.state.room);
     }
+    getProblems().then((res) => {
+      if (isError(res)) {
+        setError((res as ErrorResponse).message);
+        setProblems(null);
+      } else {
+        setProblems(res as Problem[]);
+        setError(null);
+      }
+    });
   }, [location]);
 
+  const firstProblem = problems?.[0];
+
   return (
     <div>
-      Room:
-      {' '}
-      {room ? room.roomId : 'No room joined'}
-      <Editor height="90vh" language="javascript" />
+      <Header />
+      <FlexContainer>
+        <FlexInfoBar>
+          Room:
+          {' '}
+          {room ? room.roomId : 'No room joined'}
+        </FlexInfoBar>
```

Comment:
nit: Reduce distance between this and the header? The below image seems like a big gap to me.
<img width="1278" alt="Info Bar and Header" src="https://user-images.githubusercontent.com/7987279/93164190-d32ea700-f6cd-11ea-88b1-97432e985b28.png">

Reply:
This gap is mainly due to the Header being very tall while the words are squished at the top. Since the Header affects every page, do you think I should change it?

Reply:
I'll leave that largely up to your discretion -- if you do that, make sure the other pages look OK too. If not, I'd just modify the margin of the `FlexInfoBar` (you could also make this margin negative, but I'd be careful when doing that as I'm not sure if it's good design practice).

Reply:
Not sure what's causing the space being so large (probably partly header but also maybe has something to do with padding/margin of flex container/info bar), but if you want to customize the header bar, I'd recommend creating a new, say, GameHeader file in `navigation`, which can have custom styling and text. I'm actually not sure anymore whether that should be put in the GameLayout file or whether it can be put directly in this file, because if we want to store the room id in the header (as in the mockup), then we'll need to pass that info as props to the header, which might be hard if it's in a separate file.

Reply:
I reduced the height of the room info bar; it looks slightly better but a header redesign is probably required later.